templates/zerver/api/fixtures.json

```diff
@@ -374,5 +374,45 @@
         "code": "BAD_REQUEST",
         "msg": "User not authorized for this query",
         "result": "error"
+    },
```

Comment:
Can you split adding the fixture into a first commit that we can merge without the tables?  The latter may require more iteration.

Reply:
@timabbott: Done :)